fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -2656,6 +2630,7 @@ ACTOR Future<Void> TagPartitionedLogSystem::recruitOldLogRouters(TagPartitionedL
 					req.tLogLocalities = tLogLocalities;
 					req.tLogPolicy = tLogPolicy;
 					req.locality = locality;
+					req.recoverAt = old.recoverAt + 1;
```

Comment:
Same comment here too - why are we assigning the adjusted "recoverAt" in "recoverAt"?